src/metatensor/models/utils/merge_capabilities.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple
+
+from metatensor.torch.atomistic import ModelCapabilities
+
+
+def merge_capabilities(
```

Comment:
Should there be a test for this function?

Reply:
Yes, my bad